package.json

```diff
@@ -2,12 +2,13 @@
   "dependencies": {
     "caniuse-lite": "^1.0.30001259",
     "css-loader": "^5",
-    "lux-design-system": "^4",
+    "lux-design-system": "^4.5.1",
     "postcss-cssnext": "^3.1.1",
     "sass-loader": "^10",
     "vue": "^2.6.14",
     "vue-loader": "^15.9.8",
-    "vue-template-compiler": "^2.6.14"
+    "vue-template-compiler": "^2.6.14",
+    "webpack": "^5.0.0"
```

Comment:
Does it work without adding webpack?  My understanding was that lux needs it internally, but sites that use lux shouldn't need it.

Reply:
I think it does work. I was getting a warning, so I decided to make this change. I could remove this!